src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -147,6 +144,9 @@ void CoreWorkerDirectActorTaskSubmitter::PushTask(
           waiting_reply_tasks_[actor_id].erase(task_id);
         }
         if (!status.ok()) {
+          // Note that this might be the __ray_terminate__ task, so we don't log
```

Comment:
Shouldn't __ray_terminate__ just return `Status::OK`?

Reply:
I have a fix for this in an upcoming PR...